packages/ui-extensions-react/src/surfaces/checkout/hooks/delivery-group-list-target.ts

```diff
@@ -0,0 +1,19 @@
+import type {DeliveryGroupList} from '@shopify/ui-extensions/checkout';
+
+import {useApi} from './api';
+import {useSubscription} from './subscription';
+
+/**
+ * Returns the delivery group list the extension is attached to. This hook can only be used by extensions in the following
+ * extension targets:
+ * - purchase.checkout.shipping-option-list.render-before
+ * - purchase.checkout.shipping-option-list.render-after
+ */
+export function useDeliveryGroupListTarget(): DeliveryGroupList | undefined {
+  const api = useApi<
+    | 'purchase.checkout.shipping-option-list.render-before'
+    | 'purchase.checkout.shipping-option-list.render-after'
+  >();
+
+  return useSubscription(api.target);
+}
```

Comment:
@loic-d / @DanielParpal these were automatically included in this attempt at an unstable release. But, we can hold off on these changes if needed.

Reply:
Removed